try:
  print (1 + 'st try')
except:
   print ("Isto é uma exceção")
